docs/src/pages/nextjs/appdir.mdx

```diff
@@ -70,20 +69,26 @@ export const { GET, POST } = createNextRouteHandler({
 });
 ```
 
-### Creating The UploadThing Components (optional)
+### Creating The UploadThing Components (recommended)
 
-Generating components lets you pass your generic `FileRouter` once and then have
-typesafe components everywhere, instead of having to pass the generic everytime
-you mount a component, but you can also import the components individually from
-`@uploadthing/react`.
+Generating components lets you pass your generic `FileRouter` and `url` once and
+then have typesafe components everywhere, instead of having to pass the generic
+and url everytime you mount a component, but you can also import the components
```

Comment:
```
Generating components lets you pass your generic `FileRouter` and `url` once and
then have typesafe components everywhere, instead of having to pass them
everytime you mount a component, but you can also import the components
```